CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ https://docs.docker.com/engine/deprecated/ where you can find the target removal
 * Update containerd/continuity to fix ARM 32-bit builds [moby/moby#36339](https://github.com/moby/moby/pull/36339)
 * Updated docker-on-docker build-notes. [moby/moby#35892](https://github.com/moby/moby/pull/35892)
 - Fix string type for buildargs API definition [moby/moby#36246](https://github.com/moby/moby/pull/36246)
+- Fix layer lifecycle leak [moby/moby#36338](https://github.com/moby/moby/pull/36338)
```

Comment:
Could need a description of what it's fixing (not at my computer, but can help if needed)